react-context/flow.txt

```diff
@@ -0,0 +1,18 @@
+User can sign up
+- user click on register, then the auth-form will run which prompt the user to input a username and password.
+- then the server side the post method is called then the information will be rendered on auth-from.jsx.
+- the post method then check to see if username or password is missing, if so then 401 error.
+-when password is filled out then the password is hashed and it is save to the database/data.sql.
```

Comment:
Small clarification here, data isn't actually stored in this file, this is just to hold starting data to populate your database with initially. When you use an `insert` query to save data it is stored inside the actual database application.